pkg/storaged/crypto-keyslots.jsx

```diff
@@ -606,35 +606,36 @@ function add_or_update_tang(dlg, vals, block, url, adv, old_key, passphrase) {
             .catch(request_passphrase_on_error_handler(dlg, vals, passphrase, block));
 }
 
-function edit_tang_adv(client, block, key, url, adv, passphrase) {
+export const TangKeyVerification = ({ url, adv }) => {
     const parsed = parse_url(url);
     const cmd = cockpit.format("ssh $0 tang-show-keys $1", parsed.hostname, parsed.port);
-
     const sigkey_thps = compute_sigkey_thps(tang_adv_payload(adv));
 
+    return <>
+        <p>{_("Make sure the key hash from the Tang server matches one of the following:")}</p>
+
+        <h2 className="sigkey-heading">{_("SHA256")}</h2>
+        { sigkey_thps.map(s => <p key={s} className="sigkey-hash">{s.sha256}</p>) }
+
+        <h2 className="sigkey-heading">{_("SHA1")}</h2>
+        { sigkey_thps.map(s => <p key={s} className="sigkey-hash">{s.sha1}</p>) }
```

Comment:
Is SHA1 still a thing anywhere? I can understand/tolerate it for "classic" LUKS, where users may still have ancient devices. But stratis is rather new, and IIRC isn't even fully supported in RHEL 9 (or perhaps just barely in 9.2?).

It's moved code, so not a blocker, but I'm curious.

Reply:
This is a question about Tang, not Stratis I would say. The SHA1 stuff is here for people that have old versions of tang that don't output SHA256 fingerprints. Fingerprints are computed when you run tang-show-keys so that even very very very old keys are not a problem.
So, *shrug*

Reply:
I see, thanks! (Ack to resolving)